adlik_serving/framework/manager/managed_store.cc

```diff
@@ -94,6 +99,41 @@ cub::Status ManagedStore::start(const ModelId& id) {
   return i == manageds.cend() ? startup(id) : onStartFail(id, "model was existed");
 }
 
+cub::Status ManagedStore::addModel(const std::string& modelName, const std::string& path) {
+  INFO_LOG << "add model " << modelName;
+  if (ROLE(ModelStore).exist(modelName)) {
+    INFO_LOG << modelName << " model is already exist";
+    return cub::AlreadyExists;
+  }
+  std::string targetPath = cub::paths(ROLE(ModelOptions).getBasePath(), modelName);
+  cub::Status status = cub::filesystem().copyPath(path, targetPath);
+  if (cub::isFailStatus(status)) {
+    return status;
+  }
+  ROLE(ModelStore).configOneModel(modelName);
+  ROLE(StorageLoop).once();
```

Comment:
Add a model doesn't not need to load it

Reply:
put these codes in model_operate_impl better. Let ManagedStore  not depend on StorageLoop, BoardingLoop, State Monitor better